win32ss/user/ntuser/misc.c

```diff
@@ -248,8 +248,9 @@ NtUserGetThreadState(
 
    switch (Routine)
    {
-      case THREADSTATE_GETTHREADINFO: /* FIXME: Delete this HACK */
+      case THREADSTATE_GETTHREADINFO:
          GetW32ThreadInfo();
```

Comment:
Looks like this function call is kind of a No-op.

Reply:
Deleted.